sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -357,39 +360,9 @@ For release notes and more information please visit https://aka.ms/azsdk/release
 - Removed default 60s server timeout value for retry options `tryTimeoutInMs` to avoid large blob download stream unexcepted ending.
 - Fixed an issue that when body is string with special characters, `BlockBlobULR.upload` will fail to upload.
 
-## 10.2.0-preview (2018.11)
```

Comment:
Looks like this is removed due to incorrect format.

Reply:
I will check this.

Reply:
I have found an issue in changelog modification code for older prelease versions without a prerelease number. I will file an issue and submit a PR to fix this.  @chidozieononiwu : just fyi.

Reply:
I have submitted PR https://github.com/Azure/azure-sdk-tools/pull/1331 to fix this issue.